code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -127,7 +127,9 @@
 
 /mob/living/simple_animal/drone/cogscarab/New()
 	. = ..()
-	SetLuminosity(2,1)
+	#warn a
```

Comment:
cap is strength.

or more accurately, how strong from normal (10) is the center.